deploy/terraform-aws/README.md

```diff
@@ -7,15 +7,35 @@ The Terraform config here can be used to provision a simple test cluster with ki
 ## Getting Started
 
 You need:
+
 * an AWS account configured (check that e.g. `aws iam get-user` works)
 * Terraform >= 0.12
 * aws-cli
 * jq
 
+Initialize Terraform and verify the configuration:
+
+    $ terraform init
+    [... lots of output ...]
+    $ terraform plan
+    [... lots of output ...]
+
+[Upload][0] your SSH public key to AWS, and remember its name. Make
+sure the private key for this public key is your default SSH key:
+`$HOME/.ssh/id_rsa`. Alternatively you can use an SSH key pair generated from
+AWS. In this example we’ll name our key-pair “alice-key”. Come up with
+a unique cluster name like “alice-cluster”. Finally put these values
+into a tfvars file:
+
+[0]: https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#KeyPairs:
+
+    $ cat > myenv.tfvars <<EOF
+    ssh_key_name = "alice-key"
+    cluster_name = "alice-cluster"
+    EOF
+
 You can then apply your config:
 
-    cp env.tfvars.example myenv.tfvars
-    vi myenv.tfvars # You can change settings for your cluster here.
```

Comment:
Is there a hint somewhere else in the README to use `env.tfvars.example` to create a customized `myenv.tfvars`?

Reply:
I'll add something in the documentation for that.